### PR TITLE
Add a tuned-adm command for moving devices between plugin instances

### PR DESCRIPTION
**Pull request by zacikpa**
Resolves: RHEL-15141

Until now, this was only possible by manually sending a message to TuneD via the DBus API.

### CODE REVIEW

File: man/tuned-adm.8 [hunk 1]
```diff
@@ -110,6 +110,16 @@ Enable automatic profile selection mode, switch to the recommended profile.
...
 Show current profile selection mode.
 
+.TP
+.B "instance_acquire_devices \fI[\-c | \-\-cpu\-list]\fP \fIdevices\fP \fIinstance\fP"
+Move all provided \fIdevices\fP under the given plugin \fIinstance\fP. The plugin
+instance is identified using its name from the TuneD profile.
+
+The devices are specified using a comma-separated list. When the \fI\-c\fP
+or the \fI\-\-cpu\-list\fP flag is enabled, the tool assumes that CPU devices
+are specified using a list of their numbers, such as "0,2-4", meaning that
```
Comment (yarda) [inline man/tuned-adm.8:120]: It's not just a list, it has also ranges, e.g. 2-4. The syntax [1] is called cpulist, sometimes also cpu list, CPU list or cpu-list, CPU-list, the convention is not clear. In TuneD for consistence we usually use the term "cpulist".

Recently the group modifiers were added to the syntax (:size/group) and N which we do not support yet in TuneD, but I probably wouldn't mention this maybe temporal limitation in the man.

[1] https://www.kernel.org/doc/html/next/admin-guide/kernel-parameters.html#cpu-lists

**Comment by yarda** [comment]
Thanks, maybe just update the man and built-in help, otherwise OK.

**Comment by yarda** [comment]
I was thinking a bit more about it. Maybe we could drop the -c | --cpu-list arguments and rely only on the "cpulist:" prefix the API supports and just document it. Maybe it's more consistent approach? What do you think?

**Comment by zacikpa** [comment]
> I was thinking a bit more about it. Maybe we could drop the -c | --cpu-list arguments and rely only on the "cpulist:" prefix the API supports and just document it. Maybe it's more consistent approach? What do you think?

I think that makes sense. As you say, it's more consistent, and we don't duplicate the handling of the option. I will remove it and mention the `cpulist:` prefix in the manpages.

**Review by yarda — APPROVED** [review]
LGTM, thanks.

**Pull request merged**